network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/spotlight_posts.html

```diff
@@ -3,7 +3,7 @@
 <div class="row section-spotlight section-padding">
 
   <div class="col-12">
-    <h2 class="capsule-label mb-0 ml-md-4">{% trans "Spotlight" %}</h2>
+    <h5 class="capsule-label mb-0 ml-md-4">{% trans "Spotlight" %}</h5>
```

Comment:
I checked with Tais on this - `Spotlight` is a section heading and not just for the "cover" on the left. Let's revert this back to `h2` so it can sit on the same hierarchy level as other section headings!

Reply:
```suggestion
    <h2 class="capsule-label mb-0 ml-md-4">{% trans "Spotlight" %}</h2>
```